draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -702,6 +777,9 @@
             target="RFC5905"/>. When the L-flag is 1, then it is a 64-bit
             format, otherwise it is a 32-bit short format.</t>
           </list>
+	  <!-- XXX JMH - It's necessary to discuss what happens when the F/L
+	  bits are set weirdly.  In general, it seems that the L bit is ignored
+	  when the F bit is set to 1? -->
```

Comment:
Revised the wording to the following: 

In this document, the F-Flag is set to 0. If a BGP speaker receives Service Delay Prediction Sub-TLV with F-Flag set to 1, the Service Delay Predication Value is invalid.

Reply:
While I understand your intent, I'm not clear how this text merges in.

Reply:
Tracing back the requests (on IDR list and GitHub)  that requested this Service Delay Predication Sub-TLV, the F flag is actually  to identify the types of Service Delay Predication. The Delay Predication can be in NTP Timestamp Format, or can be in abstract value, like 1-100, with larger value indicating longer delay than the smaller number. 

Do you think the following text is clear enough? 

_"Flag (F) (1 bit): Indicates whether the Service Delay is a timer value (F=0) or a relative value (F=1) where a higher value represents a longer delay"_

_"Flag (L) (1 bit): Indicates the unit of measurement for the Service Delay Prediction Value. L=0 for 64-bit NTP Timestamp Format (valid only when F-flag is set to 0). L=1 for milliseconds (valid only when F-flag is set to 1)"_

Reply:
The above text, and what's in the current branch, is most of the way there.

What you're missing is describing for F=0, what do you do with L?  Ignore it?  If so, basically say "F=0... L is ignored".